core/clingutils/CMakeLists.txt

```diff
@@ -129,8 +129,9 @@ endforeach()
 # of that file in build-time versus run-time (different length etc).
 # Capture their build-time version here, and inject it into runtime.
 foreach(file wchar.h bits/stat.h bits/time.h)
-  if(EXISTS /usr/include/${file})
-    list(APPEND copy_commands COMMAND ${CMAKE_COMMAND} -E copy /usr/include/${file} ${CMAKE_BINARY_DIR}/etc/cling/lib/clang/${CLANG_RESOURCE_DIR_VERSION}/include/${file})
+  if(EXISTS ${DEFAULT_SYSROOT}/usr/include/${file})
```

Comment:
@Axel-Naumann is this ok for you? I think this whole foreach should be removed, but that not being possible, we should at least try to copy the headers from the right place when sysroot != /. For example, on Mac OS, this probably does the wrong thing, since in many cases `/usr/include` no longer exists anyway.

Reply:
I will merge this since `${DEFAULT_SYSROOT}` is normally empty, so it doesn't affect the usual mode of compilation for regular users. We can revert later if needed.